### ISSUE TITLE
Folder is displayed as empty when clicked from search results

### ISSUE THREAD

**Issue by TravisYeah** [BloopAI/bloop#634]
**Describe the bug**
Folder is displayed as empty when clicked from search results.

**Expected behavior**
The folder should display the three files shown in the video below.

**To Reproduce**

I used the JupyterLab GitHub repository and searched `path:packages/console/style`.

**Screenshots or output**

https://github.com/BloopAI/bloop/assets/36907905/91e3ac65-3636-4f3c-8e02-ab8310571fe7

**Comment by ggordonhall** [comment]
Can you link the GitHub repository that you have indexed?

**Comment by rsdy** [comment]
Can you please confirm if your host platform is Windows? I notice the `\` at the end of your prompt in the video. If that's the case, this is probably a Windows-specific thing.

**Comment by TravisYeah** [comment]
Yes I am on Windows. Here is the repository https://github.com/jupyterlab/jupyterlab.
